presto-parser/src/main/antlr4/com/facebook/presto/sql/parser/SqlBase.g4

```diff
@@ -55,6 +55,11 @@ statement
     | ANALYZE qualifiedName (WITH properties)?                         #analyze
     | CREATE (OR REPLACE)? VIEW qualifiedName AS query                 #createView
     | DROP VIEW (IF EXISTS)? qualifiedName                             #dropView
+    | CREATE (OR REPLACE)? FUNCTION functionName=qualifiedName
+      '(' (parameterDeclaration (',' parameterDeclaration)*)? ')'
+      RETURNS returnType=type
+      routineCharacteristics
+      RETURN body=expression                                           #createFunction
```

Comment:
Let's make this future proof. `RETURN should be part of the body`. `RETURN expression` is one type of function body. Otherwise there will be major structural changes when we introduce other languages, or more complex SQL functions.